app.coffee

```diff
@@ -141,7 +141,8 @@ if Settings.shutdownDrainTimeWindow?
 	if Settings.errors?.catchUncaughtErrors
 		process.removeAllListeners('uncaughtException')
 		process.on 'uncaughtException', (error) ->
-			if error.code == 'EPIPE'
+			if ['EPIPE', 'ECONNRESET'].includes(error.code)
+				Metrics.inc('disconnected_write')
```

Comment:
Could set the error code as a label:
Metrics.inc('disconnected_write', 1, {status: error.code})